tests/conftest.py

```diff
@@ -7,17 +7,21 @@
 from unittest.mock import Mock
 from ruamel.yaml import YAML
 from conda_vendor.conda_vendor import (
-        get_conda_platform,
-        get_lock_spec_for_environment_file,
-        solve_environment,
-        get_fetch_actions,
-        )
-from conda_lock.conda_solver import DryRunInstall, VersionedDependency, FetchAction
+    solve_environment,
+)
+from conda_lock.conda_solver import (
+    DryRunInstall,
+    VersionedDependency,
+    FetchAction,
+)
 from conda_lock.src_parser import LockSpecification
+from conda_lock.src_parser.environment_yaml import parse_environment_file
+
+
 # Test Fixtures
 
 
-# minimal conda environment containing 
+# minimal conda environment containing
 # only pinned python with channel main
```

Comment:
What does the scope="function" do?

Reply:
leaves the fixture alive as long as the function is active. 
https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session